.ruby-version

```diff
@@ -1 +1 @@
-2.5.0
+3.3.3
```

Comment:
Perhaps just specify `3.3`, since there's already a stable 3.3.5 and in general we want those patches